### PR TITLE
Db connection

### PR DESCRIPTION
**Pull request by SmartCode18**

### CODE REVIEW

File: src/main/java/domain/HelloWorldWhileImpl.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package domain;
+
+public class HelloWorldWhileImpl {
```
Comment (fairwinds0099) [inline src/main/java/domain/HelloWorldWhileImpl.java]: pleser emove this class

File: src/main/java/domain/Pojos/Address.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
+package domain.Pojos;
+
+public class Address {
+    private String id;
+    private String streetName;
+    private String city;
+    private String zipCode;
```
Comment (fairwinds0099) [inline src/main/java/domain/Pojos/Address.java]: too early remove this Class

File: src/main/java/domain/enums/Query.java [hunk 3]
```diff
@@ -2,7 +2,8 @@
 
 //TODO to be updated
 public enum Query {
-    REPID("SELECT FIRST_NAME FROM REPRESENTATIVES WHERE REP_ID = ?");
+    REPBYID("SELECT CITY FROM ADDRESS WHERE ID = ?"),
```
Comment (fairwinds0099) [inline src/main/java/domain/enums/Query.java]: remmove enum

File: src/main/java/domain/helpers/QueryHandlingHelpers.java [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+import domain.enums.Query;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class QueryHandlingHelpers {
+
+       private DbSingleton instance = DbSingleton.getInstance();
```
Comment (fairwinds0099) [inline src/main/java/domain/helpers/QueryHandlingHelpers.java]: remove singleton

File: src/main/java/domain/helpers/QueryHandlingHelpers.java [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+       private Connection connection = instance.getConnection();
+
+       protected Representatives findRepresentative(String repId){
+              Representatives rep = new Representatives();
+              try (PreparedStatement statement = connection.prepareStatement(Query.REPBYID.getQuery())){
+                     statement.setString(1,repId);
```
Comment (fairwinds0099) [inline src/main/java/domain/helpers/QueryHandlingHelpers.java]: remove setters and getters

File: src/main/java/domain/helpers/QueryHandlingHelpers.java [hunk 6]
```diff
@@ -0,0 +1,45 @@
+package domain.helpers;
```
Comment (fairwinds0099) [inline src/main/java/domain/helpers/QueryHandlingHelpers.java]: remove helpers

File: src/main/java/tests/HelloWorldTest.java [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+import domain.HelloWorldForImpl;
+import domain.HelloWorldWhileImpl;
+import domain.helpers.QueryHandlingHelpers;
+import org.testng.annotations.Test;
+
+
+public class HelloWorldTest {
+
+    HelloWorldWhileImpl whileHello = new HelloWorldWhileImpl();
+    HelloWorldForImpl forHello = new HelloWorldForImpl();
```
Comment (fairwinds0099) [inline src/main/java/tests/HelloWorldTest.java]: remove everything related to hellow orld

File: pom.xml [hunk 8]
```diff
@@ -94,10 +94,26 @@
...
+            <artifactId>rest-assured</artifactId>
+            <version>3.1.1</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
+            <scope>test</scope>
+        </dependency>
```
[the remark this answers is not in the document]
Comment (SmartCode18) [reply pom.xml]: Download ojdbc7.jar file from Oracle website
Install this file using the command prompt with below command:
mvn install:install-file -Dfile=Downloads//ojdbc7.jar -DgroupId=com.oracle -DartifactId=ora-jdbc -Dversion=8 -Dpackaging=jar
Add the dependency to pom.xml file with updated data.

**Comment by SmartCode18** [comment]
Once we upgrade the jdbc version, previous version will be kept in .m2 folder, as well.

**Comment by SmartCode18** [comment]
drwxr-xr-x  38 ike  staff  1292 Oct 18 22:10 .
drwxr-xr-x   3 ike  staff   102 Sep 16 12:38 ..
drwxr-xr-x   3 ike  staff   102 Sep 16 12:38 antlr
drwxr-xr-x   3 ike  staff   102 Oct 18 22:10 aopalliance
drwxr-xr-x   3 ike  staff   102 Sep 16 12:38 avalon-framework
drwxr-xr-x   3 ike  staff   102 Sep 16 12:38 backport-util-concurrent
drwxr-xr-x   3 ike  staff   102 Sep 16 12:38 classworlds
drwxr-xr-x  10 ike  staff   340 Oct 18 22:34 com
drwxr-xr-x   3 ike  staff   102 Sep 16 12:38 commons-beanutils
drwxr-xr-x   3 ike  staff   102 Sep 16 12:38 commons-chain
drwxr-xr-x   3 ike  staff   102 Sep 16 12:38 commons-cli
drwxr-xr-x   3 ike  staff   102 Sep 16 12:38 commons-codec
drwxr-xr-x   3 ike  staff   102 Sep 16 12:38 commons-collections
drwxr-xr-x   3 ike  staff   102 Sep 16 12:38 commons-digester
drwxr-xr-x   3 ike  staff   102 Sep 16 12:38 commons-io
drwxr-xr-x   3 ike  staff   102 Sep 16 12:38 commons-lang
drwxr-xr-x   4 ike  staff   136 Sep 16 12:38 commons-logging
drwxr-xr-x   3 ike  staff   102 Sep 16 12:38 commons-validator
drwxr-xr-x   3 ike  staff   102 Sep 16 12:38 dom4j
drwxr-xr-x   5 ike  staff   170 Oct 14 21:37 io
drwxr-xr-x   4 ike  staff   136 Oct 14 21:37 javax
drwxr-xr-x   3 ike  staff   102 Oct 14 21:37 jdom
drwxr-xr-x   3 ike  staff   102 Oct 14 21:37 jtidy
drwxr-xr-x   3 ike  staff   102 Sep 16 12:38 junit
drwxr-xr-x   3 ike  staff   102 Sep 16 12:38 log4j
drwxr-xr-x   3 ike  staff   102 Sep 16 12:38 logkit
drwxr-xr-x   3 ike  staff   102 Oct 14 21:37 net
drwxr-xr-x   3 ike  staff   102 Oct  2 00:07 oracle
drwxr-xr-x  21 ike  staff   714 Oct 18 22:09 org
drwxr-xr-x   3 ike  staff   102 Sep 16 12:38 oro
drwxr-xr-x   4 ike  staff   136 Oct 14 21:37 plexus
drwxr-xr-x   3 ike  staff   102 Sep 16 12:38 sslext
drwxr-xr-x   3 ike  staff   102 Sep 16 12:38 xerces
drwxr-xr-x   3 ike  staff   102 Sep 16 12:38 xml-apis
drwxr-xr-x   3 ike  staff   102 Oct 14 21:37 xml-resolver
drwxr-xr-x   3 ike  staff   102 Oct 14 21:37 xmlpull
drwxr-xr-x   3 ike  staff   102 Oct 14 21:37 xmlunit
drwxr-xr-x   3 ike  staff   102 Oct 14 21:37 xpp3
Aytekin-MBP:repository ike$ cd com
Aytekin-MBP:com ike$ cd siemens
Aytekin-MBP:siemens ike$ cd maven-restassured-jdbc
Aytekin-MBP:maven-restassured-jdbc ike$ ls -la
total 8
drwxr-xr-x  5 ike  staff  170 Oct 18 22:35 .
drwxr-xr-x  3 ike  staff  102 Oct 18 22:34 ..
drwxr-xr-x  5 ike  staff  170 Oct 18 22:34 0.1
drwxr-xr-x  5 ike  staff  170 Oct 18 22:35 0.2
-rw-r--r--  1 ike  staff  338 Oct 18 22:37 maven-metadata-local.xml
Aytekin-MBP:maven-restassured-jdbc ike$ cd 0.1
Aytekin-MBP:0.1 ike$ ls -la
total 32
drwxr-xr-x  5 ike  staff   170 Oct 18 22:34 .
drwxr-xr-x  5 ike  staff   170 Oct 18 22:35 ..
-rw-r--r--  1 ike  staff   206 Oct 18 22:34 _remote.repositories
**_-rw-r--r--  1 ike  staff  6424 Oct 18 22:34 maven-restassured-jdbc-0.1.jar_**
-rw-r--r--  1 ike  staff  1774 Oct 18 22:31 maven-restassured-jdbc-0.1.pom
Aytekin-MBP:0.1 ike$ 

If we change the version of the project, every changes on the jar file will be kept in .m2 folder .

**Comment by fairwinds0099** [comment]
perfect

File: src/main/java/tests/DbUtils.java [hunk 9]
```diff
@@ -16,12 +16,36 @@ public static void main (String [] args) throws SQLException {
...
+
+        ResultSet rsltRe = stRead.executeQuery(queryRead);
+
+        System.out.println (rsltRe);
+
+        String queryUpdate = "UPDATE ADDRESS SET CITY = 'Miami', ZIPCODE='22222' WHERE ID='7'";
+
+        Statement stUpdate = conn.prepareStatement(queryUpdate);
+
+        ResultSet rsltUp = stUpdate.executeQuery(queryUpdate);
```
Comment (fairwinds0099) [inline src/main/java/tests/DbUtils.java]: do we need ResultSet?

File: src/main/java/tests/DbUtils.java [hunk 10]
```diff
@@ -16,12 +16,36 @@ public static void main (String [] args) throws SQLException {
...
-        String query = "INSERT INTO ADDRESS (ID, STREETNAME, CITY, ZIPCODE, STATE)" +
-                "VALUES(7, '333 State St', 'Mc Lean', '44044', 'NJ')";
+        String queryCreate = "INSERT INTO ADDRESS (ID, STREETNAME, CITY, ZIPCODE, STATE)" +
+                "VALUES(7, '311 True St', 'Richmond', '54544', 'VA')";
 
-        Statement statement = conn.prepareStatement(query);
+        Statement stCreate = conn.prepareStatement(queryCreate);
 
-        ResultSet resultSet = statement.executeQuery(query);
+        ResultSet rsltCr = stCreate.executeQuery(queryCreate);
```
[the remark this answers is not in the document]
Comment (SmartCode18) [reply src/main/java/tests/DbUtils.java]: I just removed Resultset objects and can still run the test successfully. 
For printing object, I just tried if it is possible. I might not have the fundamentals of java yet...

File: src/main/java/tests/DbUtils.java (hunk 9 above)
Comment (SmartCode18) [reply src/main/java/tests/DbUtils.java]: It's not a must as I observe.

**Pull request closed without merging**